jekyll/_cci2/monitoring.adoc

```diff
@@ -54,7 +60,69 @@ https://www.nomadproject.io/docs/telemetry/metrics.html#job-metrics[Nomad job me
 |===
 
 When the Nomad metrics container is running normally, no output will be written to standard output or standard error. Failures will elicit a message to standard error.
+// What is meant by Standard Output and Standard Error?
+
+=== CircleCI Metrics
+// This title isn't write I don't think, but not sure what to call them?
```

Comment:
I mean they are Metrics from CircleCI, is there some question about whether they should or shouldn't apply?

Reply:
The reason I put this question is that there are metics that start `circle.` outside this section, so aren;t they CircleCI metics too? Seems we're to have that heading for just this set?

Reply:
Also see comment below the table - there are some nomad metrics here which look like they should be in the table above?